app/src/test/java/com/codesoom/assignment/controllers/HelloControllerTest.java

```diff
@@ -0,0 +1,18 @@
+package com.codesoom.assignment.controllers;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * HelloController에 대해 유닛테스트를 실행한다.
+ */
```

Comment:
이 주석은 생략하고 `@DisplayName`을 사용해 봅시다.
다음과 같이 사용할 수 있을 거에요.

```suggestion
@DisplayName("HelloController")
```

위와 같이 작성한다는 것은 이 클래스는 `HelloController`를 테스트하는 코드를 갖고 있다는 것을 표현하는 것이에요.

Reply:
네 이해했습니다 훨씬 깔끔하네요!